### PR TITLE
Setup code coverage and badges

### PR DESCRIPTION
**Pull request by plule-ansys**
Add the badges (the list comes mostly from the pymapdl repo), and setup the code coverage

### CODE REVIEW

**Comment by plule-ansys** [comment]
The coverage badge is targetting the `main` branch with no result yet, but it will look like this:

report

**Comment by plule-ansys** [comment]
I will wait for [this pr approval](https://github.com/pyansys/ansys-templates/pull/69) for the PyAnsys logo

**Review by baslas — APPROVED** [review]

**Pull request merged**
